src/styles/blocks/star.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
```

Comment:
Consider using a variable for the star size (height and width) and margin-right value to make it easier to maintain and update in the future. This can be defined in src/styles/utils/variables.scss and then imported here.